pkg/mover/chart.go

```diff
@@ -126,9 +142,11 @@ type ChartMover struct {
 	chartChanges            []*internal.RewriteAction
 	sourceContainerRegistry internal.ContainerRegistryInterface
 	targetContainerRegistry internal.ContainerRegistryInterface
+	targetOfflineTar        string
 	chart                   *chart.Chart
 	logger                  Logger
 	retries                 uint
+	rawHints                []byte
```

Comment:
Could you add a comment explaining what's the expected content of this?

Reply:
Done